lib/engines/object_store/app/views/object_store/content_blocks/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>New <%= @content_block.block_type %></h1>
+
+<%= form_with(model: @content_block, url: "/object-store/content_blocks", local: true) do |form| %>
```

Comment:
I had to hardcode the path here because otherwise it would default to `/object-store/assets/whitehall?action=index&controller=object_store%2Fcontent_blocks`... I'm not sure what middleware was causing that

Reply:
I think we might have more predictable results by replacing:

```
ObjectStore::Engine.routes.draw do
```

with:

```
Whitehall::Application.routes.draw do
```

in the routes.rb file

Reply:
ah yep, that seems to have fixed things